vmslave-config/ansible/roles/install-package-for-ci/tasks/main.yml

```diff
@@ -46,11 +46,17 @@
     executable: /usr/bin/expect
   when: ovftool_check.rc !=0
 
+- name: Check if apache2 is installed
```

Comment:
Every vmslave has it's own static file server, though this server is not used in any pipeline now.